submissions/natashafir/post-oop/index.js

```diff
@@ -0,0 +1,60 @@
+class Inhabitants {
+    constructor(species, name, gender, legs, hands, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.legs = legs;
+        this.hands = hands;
+        this.saying = saying;
+    }
+
+    sayHi() {
+        return "My name is " + this.name + ". I'm " + this.species + ". My gender is " + this.gender +
+            ". I have " + this.legs + " legs and " + this.hands + " hands. " + this.saying
```

Comment:
Quite a lengthy string composition. Imagine, we add a dozen of props more.
How about splitting it in logical groups in an array and then join them with `Array.join`?